wsidicom/instance.py

```diff
@@ -977,11 +977,11 @@ def _get_frame_information(
             tiled_sizes.add(image_data.tiled_size)
 
         focal_planes_sparse_by_optical_path = any(
```

Comment:
This looks like it can be simplified by using set.isdisjoint(), https://docs.python.org/3/library/stdtypes.html#frozenset.isdisjoint